src/widget/OktaSignIn.js

```diff
@@ -12,7 +12,7 @@ var OktaSignIn = (function () {
      * @param callback - callback function invoked with 'true'/'false' as the argument.
      */
     function checkSession(callback) {
-      authClient.existingSession().then(callback);
+      authClient.session.exists().then(callback);
```

Comment:
This resolves a promise with a boolean of whether the session exists. It has to be a promise because we make an api call for the check